examples/ffva/bsp_config/XK_VOICE_L71/platform/platform_start.c

```diff
@@ -45,15 +45,18 @@ static void flash_start(void)
 
 static void i2c_master_start(void)
 {
+#if !appconfI2C_DFU_ENABLED
```

Comment:
Two questions:

- What is the reason to exclude this functionality if I2C DFU is enabled?  I can make a guess, but I would like to know to avoid reaching an incorrect conclusion.
- If I2C DFU is enabled and the `i2c_master_start()` function is called, it silently does nothing.  What is the effect on the application in this case?

Reply:
DFU uses I2C slave, and the DAC uses I2C master, and I was worried it can cause issues having two masters on the bus.

I will remove that #if, as it shouldn't be a problem if we use I2C master to configure the DAC just at initialization.

Reply:
My previous comment was not correct. DFU over i2C requires I2C slave to send/receive the control messages. I2C master is used to configure the DAC and to send some messages when a wakeword/voice command is detected. We can assume that the DAC is configured at startup, but the messages can be sent while a DFU is ongoing.
We don't have a mechanism to switch between I2C master and slave, so at the moment we need to assume that if DFU over I2C is enabled, we can support only I2C slave.